tree/ntuple/v7/src/RFieldBase.cxx

```diff
@@ -20,16 +20,6 @@
 
 namespace {
 
-/// Resolve usings and typedefs
-std::string GetResolvedTypeName(const std::string &typeName)
-{
-   // The following types are asummed to be resolved; thus, do not perform `typedef` resolution on those
-   if (typeName.substr(0, 5) == "std::" || typeName.substr(0, 25) == "ROOT::RNTupleCardinality<")
-      return typeName;
-
-   return TClassEdit::ResolveTypedef(typeName.c_str());
-}
-
```

Comment:
Can you add some more details to the commit message what was wrong / what is fixed? It's probably in some of the discussion threads of this PR, but I can't find it...